app/models/comment.rb

```diff
@@ -146,7 +147,7 @@ def parent_type
   def id_code_generated
     # 26 is the conversion base
     # eg. 1000.to_s(26) would be "1cc"
-    id.to_s(26)
+    id.to_fs(26)
```

Comment:
I don't think this change makes sense — `to_s` is from Ruby and is for explicitly converting an integer into a string with the supplied base, `to_fs` is from ActiveSupport and is mostly used for i18n with Date but also with Numeric for `:human` and `:phone` and `:currency`. It _works_, but I think the original `to_s` call is the better way for this usage.

```suggestion
    id.to_s(26)
```

Reply:
Yes, the part of code in my fork is quite old, and offended a Rubocop rule which seems no longer there https://github.com/rubocop/rubocop-rails/pull/869.